app/models/manager_refresh/inventory/core.rb

```diff
@@ -0,0 +1,56 @@
+module ManagerRefresh::Inventory::Core
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def provider_module
+      @provider_module ||= ManageIQ::Providers::Inflector.provider_module(self)
+    end
+
+    def has_authentications(options = {})
```

Comment:
ah, I see, this became a module

so this is more ManagerRefresh::Inventory::Core::AutomationManager ? My point is that I want a file per manager, not 1 Core with 100 methods :-)